core/tests/command/test_command.py

```diff
@@ -63,6 +63,29 @@ def test_create():
     )
 
 
+def test_change_action():
```

Comment:
This test is a little odd; it's only really validating that the action can be cleared, which I think would have always been true.